app/isomorphic/components/layouts/footer/index.js

```diff
@@ -1,49 +1,31 @@
 import React from "react";
-import PT from "prop-types";
-import { connect } from "react-redux";
-import get from "lodash/get";
-import { Link } from "@quintype/components";
 import "./styles.m.css";
+import logo from "./logo.png";
+import assetify from "@quintype/framework/assetify";
 
-const FooterBase = ({ footerLinks }) => (
-  <div styleName="footer">
-    {footerLinks.map(
-      item =>
-        item.isExternalLink ? (
-          <a
-            href={item.completeUrl}
-            target="_blank"
-            rel="noopener noreferrer"
-            styleName="link"
-          >
-            {item.title}
-          </a>
-        ) : (
-          <Link href={item.completeUrl} styleName="link">
-            {item.title}
-          </Link>
-        )
-    )}
-  </div>
-);
+const Footer = () => {
+  const categories = ["First", "Second", "Third", "Forth", "Fifth"];
+  const sections = ["First", "Second", "Third"];
```

Comment:
can we use slice to use the categories array, instead of creating a new array?

Reply:
Right agreed but the data need not be the same. it just happened the the data is same here